akka-http-core/src/main/scala/akka/http/impl/engine/rendering/HttpResponseRendererFactory.scala

```diff
@@ -33,22 +33,22 @@ private[http] class HttpResponseRendererFactory(serverHeader: Option[headers.Ser
     }
 
   // as an optimization we cache the Date header of the last second here
-  @volatile private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
-
+  private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
   private def dateHeader: Array[Byte] = {
-    var (cachedSeconds, cachedBytes) = cachedDateHeader
-    val now = System.currentTimeMillis
-    if (now - 1000 > cachedSeconds) {
```

Comment:
Scrap my comment from above.
What is needed is replacing the `-` here with `/`.